modules/artifact/src/main/java/org/mule/module/artifact/classloader/IBMMQResourceReleaser.java

```diff
@@ -93,6 +100,9 @@ public void release() {
     LOGGER.debug("Releasing IBM MQ resources - Removes the static references held by the TraceController Class.");
     cleanPrivateStaticFieldForClass(IBM_MQ_TRACE_CLASS, "traceController");
 
+    LOGGER.debug("Releasing IBM MQ resources - Removes JMSCCThreadPoolMaster threads.");
+    disposeMQThreads();
```

Comment:
This disposal which was originally in the `mule-ibm-mq-connector` had [a property to disable it](https://github.com/mulesoft/mule-ibm-mq-connector/blob/f550d3a865f9375a8885d75560183b9d193047bf/src/main/java/com/mulesoft/connectors/ibmmq/internal/connection/IBMMQConnectionProvider.java#L281). We should probably keep that property to prevent breaking backwards.